src/pages/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { getAllCommentsFromArticle }  from "../Api";
+import { useParams } from "react-router-dom";
+
+const CommentsList = () => {
+  const { article_id } = useParams()
+const [comments, setComments] = useState([]);
+
+useEffect(() => {
+  getAllCommentsFromArticle(article_id).then((commentsFromApi) => {
+    setComments(commentsFromApi)
+  });
+}, [article_id]);
+
+return (
+  <>
+  <div className="comments-container">
+  <h2>These are the comments for the article</h2>
+  <ul>
+    {comments.map((comment) => (
+      <li key={comment.comment_id} className="comment-card">
+        <p className="body">{comment.body}</p>
+        <p className="author"> Author:{comment.author}</p>
+        <p className="date">Date:{comment.created_at}</p>
```

Comment:
I would format this created at, you can either split by the first 10 characters. Or you can use a function like `<p>{new Date(created_at).toLocaleDateString()}<p>`